### PR TITLE
OPHJOD-261: Add RadioButton & RadioButtonGroup components

### PR DESCRIPTION
**Pull request by ketsappi**
## Description

- Add RadioButton
- Add RadioButtonGroup

## Related JIRA ticket

https://jira.eduuni.fi/browse/OPHJOD-261

### CODE REVIEW

File: lib/components/RadioButton/RadioButton.test.tsx [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  cleanup();
+});
+
+const Wrapper = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <RadioButtonGroup label="" value="" onChange={vi.fn()}>
+      {children}
+    </RadioButtonGroup>
+  );
+};
```
Comment (ketsappi) [inline lib/components/RadioButton/RadioButton.test.tsx:18]: Wrapper as RadioButton requires the group, so group is same for all tests.

File: lib/components/RadioButton/RadioButton.tsx [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+const UncheckedIcon = () => {
+  return (
+    <svg width="32" height="32" viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
+      <g>
+        <circle cx="15.6992" cy="16.5333" r="15.6992" fill="white" />
+      </g>
+    </svg>
+  );
+};
```
Comment (ketsappi) [inline lib/components/RadioButton/RadioButton.tsx:41]: Later these should come from separate icon-files where others are located also.

File: lib/components/RadioButton/RadioButton.test.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import '@testing-library/jest-dom/vitest';
+import React from 'react';
+
+afterEach(() => {
+  cleanup();
+});
+
+const Wrapper = ({ children }: { children: React.ReactNode }) => {
+  return (
+    <RadioButtonGroup label="" value="" onChange={vi.fn()}>
```
Comment (sauanto) [inline lib/components/RadioButton/RadioButton.test.tsx:14]: I think it is better to add some `label` and `value` than an empty string.

**Review by sauanto — APPROVED** [review]

File: lib/components/RadioButton/RadioButton.test.tsx (hunk 3 above)
Comment (ketsappi) [reply lib/components/RadioButton/RadioButton.test.tsx:14]: I can add those. As this wrapper is not under the actual test I did not consider it crucial.

**Pull request merged**